src/main/kotlin/no/nav/faktureringskomponenten/controller/dto/FakturaserieRequestDto.kt

```diff
@@ -41,17 +44,17 @@ data class FakturaserieRequestDto(
     val referanseNAV: String,
 
     @field:Schema(
-        description = "Informasjon om hva bruker betaler",
-        example = "Trygdeavgift",
+        description = "Informasjon om hvilket tema fakturaen gjelder",
+        example = "TRY",
     )
-    @field:NotBlank(message = "Du må oppgi fakturaGjelder")
-    val fakturaGjelder: String,
+    @field:NotNull(message = "Du må oppgi fakturaGjelder")
+    val fakturaGjelder: FakturaGjelder,
```

Comment:
Burde være DTO her som på FakturaserieIntervallDto for å hindre kobling direkte mellom domene og API